queries/gdscript/highlights.scm

```diff
@@ -30,7 +30,8 @@
   (#match? @type "^[A-Z]"))
 ((identifier) @constant
   (#match? @constant "^[A-Z][A-Z_0-9]*$"))
-((identifier) @type . ".")
+((identifier) @type
+  (#match? @type "^[A-Z][A-Z_0-9]*$") . (_))
```

Comment:
Are caps only objects in gdscript a type, like the `OS` example you showed?

Reply:
No, this is just an override to avoid highlighting types like `OS` as a constant. A have a different rule for a type upper.

Reply:
:+1: